civis/polling.py

```diff
@@ -38,6 +38,19 @@ class PollableResult(futures.Future):
     polling_interval : int or float
         The number of seconds between API requests to check whether a result
         is ready.
+
+    Examples
+    --------
+    >>> client = civis.APIClient()
+    >>> database_id, sql, preview_rows, cred_id = 999, "SELECT 1", 10, 999
```

Comment:
Would it be useful to use `client.get_database_id` and `client.get_database_credential_id` in this example, or is that out of scope?